docs/baseband/dada/index.rst

```diff
@@ -20,9 +15,7 @@ Usage
 All files should be opened using :func:`~baseband.dada.open`.  Single files can
 be opened in binary mode, which provides a normal file reader but extended with
 methods to read a :class:`~baseband.dada.DADAFrame`.  For dada files, which
-consist of just a single header and payload, such frames contain all the data.
-
-::
+consist of just a single header and payload, such frames contain all the data.::
```

Comment:
Sphinx builds this as " ...such frames contain all the data.: ", with a trailing colon, which is why I added the indented block flag after a newline.

Reply:
In that case, I think just omit the period - a colon is fine.